cms/templates/widgets/footer.html

```diff
@@ -11,16 +11,22 @@
         <p>&copy; ${settings.COPYRIGHT_YEAR} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
       </div>
 
-      <nav class="nav-peripheral">
-        <ol>
-          <li class="nav-item nav-peripheral-tos">
-            <a data-rel="edx.org" href="${marketing_link('TOS')}">${_("Terms of Service")}</a>
-          </li>
-          <li class="nav-item nav-peripheral-pp">
-            <a data-rel="edx.org" href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
-          </li>
-        </ol>
-      </nav>
+      % if (marketing_link('TOS') and marketing_link('TOS') != '#') or (marketing_link('PRIVACY') and marketing_link('PRIVACY') != '#'):
```

Comment:
I don't think you need to have this - having the next part be an empty `<nav>` won't hurt anything